npm install failed with "No compatible version found: node-sass@'>=3.0.0-pre <4.0.0'"

```
npm ERR! notarget No compatible version found: node-sass@'>=3.0.0-pre <4.0.0'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.9.4-rc1","0.2.0","0.2.1","0.2.2","0.2.3","0.2.4","0.2.5","0.2.6","0.3.0","0.4.0","0.4.1","0.4.2","0.4.3","0.4.4","0.5.0","0.5.1","0.5.2","0.5.3","0.5.4","0.6.0","0.6.1","0.6.2","0.6.3","0.6.4","0.6.5","0.6.6","0.6.7","0.7.0-alpha","0.7.0","0.8.0","0.8.1","0.8.2","0.8.3","0.8.4","0.8.5","0.8.6","0.9.0","0.9.1","0.9.2","0.9.3","0.9.4-rc1","0.9.4","0.9.5-rc1","0.9.5","0.9.6","1.0.0","1.0.1","1.0.2-alpha","1.0.2","1.0.3","1.1.0","1.1.1","1.1.2","1.1.3","1.1.4","1.2.0","1.2.1","1.2.2","1.2.3","2.0.0-beta","2.0.0","2.0.1","3.0.0-alpha.0","3.0.0-beta.2","3.0.0-beta.3","3.0.0-beta.4","2.1.0","2.1.1","3.0.0-beta.5"]
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "gulp-coffee": "^2.2.0",
     "gulp-plumber": "^0.6.6",
     "gulp-react-jade": "mizchi/gulp-react-jade#9cfde076190054c29ec4922fc3fe0a178f17b897",
-    "gulp-sass": "Keats/gulp-sass#81e4f7dc70f9fc7488b8a65f96f448f4f9538d9d",
+    "gulp-sass": "Keats/gulp-sass#801b1a51265ef3675979cfcf647cc6a90bc0f52f",
     "gulp-shell": "^0.2.11",
     "gulp-watchify": "^0.3.0",
     "jsdom": "^4.0.1",

